src/main/sensors/gyro.c

```diff
@@ -681,9 +683,10 @@ STATIC_UNIT_TESTED void performGyroCalibration(gyroSensor_t *gyroSensor, uint8_t
                 gyroSetCalibrationCycles(gyroSensor);
                 return;
             }
-            gyroSensor->gyroDev.gyroZero[axis] = (gyroSensor->calibration.sum[axis] + (gyroCalculateCalibratingCycles() / 2)) / gyroCalculateCalibratingCycles();
+            gyroSensor->gyroDev.gyroZero[axis] = gyroSensor->calibration.sum[axis] / gyroCalculateCalibratingCycles();
         }
     }
+    gyro_offset = (float)gyroConfig()->gyro_offset/1000;
```

Comment:
this still doesnt look like the correct place. 
performGyroCalibration gets called in a loop and this caluclation allways is the same.
this should be in a init function and not inside a loop